bundler/minify.js

```diff
@@ -1,69 +1,33 @@
 "use strict"
 
-var http = require("https")
-var querystring = require("querystring")
 var fs = require("fs")
+var UglifyES = require("uglify-es")
 
-module.exports = function(input, output, options, done) {
-	function minify(input, output) {
-		var code = fs.readFileSync(input, "utf8")
+module.exports = function(filePath, options) {
+	function minify(filePath) {
+		var original = fs.readFileSync(filePath, "utf8"),
+			uglified = UglifyES.minify(original),
+			compressed = uglified.code
 
-		var data = {
-			output_format: "json",
-			output_info: ["compiled_code", "warnings", "errors", "statistics"],
-			compilation_level: options.advanced ? "ADVANCED_OPTIMIZATIONS" : "SIMPLE_OPTIMIZATIONS",
-			warning_level: "default",
-			output_file_name: "default.js",
-			js_code: code,
+		if (compressed) {
```

Comment:
Are there more cases then success (compress) or error?

I would rather ditch the whole else part. Like so
```suggestion
		if (uglified.error) {
			throw new Error(uglified.error)
		}
		fs.writeFileSync(filePath, compressed, "utf8")
		return {original: original, compressed: compressed}
```

Reply:
this what this function either returns the upper object or throws, so [this](https://github.com/MithrilJS/mithril.js/pull/2285/files#diff-856ea5721bc1c132715b0615f3cbb105R38) is always true and can therefore be kicked.

Reply:
@StephanHoyer Nope, there are no cases other than compress or error. This is in now, thanks.